client/client.go

```diff
@@ -19,7 +20,7 @@ type client struct {
 
 // New returns a new instance of a Spacelift Client.
 func New(wraps *http.Client, session session.Session) Client {
```

Comment:
Could we perhaps reuse my fork of the GraphQL client that we're already using in our Terraform provider?
https://github.com/marcinwyszynski/graphql

Reply:
Sure